.evergreen/mongodl.py

```diff
@@ -800,9 +800,9 @@ def _latest_build_url(
     edition: str,
     component: str,
     branch: "str|None",
-) -> str:
+) -> tuple[str, str]:
```

Comment:
```suggestion
) -> str:
```
Still just `str` for `_latest_build_url`.